internal/distro/distro.go

```diff
@@ -35,15 +35,16 @@ var (
 	oemLookasideDir = "/usr/share/oem"
 
 	// Helper programs
-	chrootCmd   = "/usr/bin/chroot"
-	groupaddCmd = "/usr/sbin/groupadd"
-	idCmd       = "/usr/bin/id"
-	mdadmCmd    = "/usr/sbin/mdadm"
-	mountCmd    = "/usr/bin/mount"
-	sgdiskCmd   = "/usr/sbin/sgdisk"
-	udevadmCmd  = "/usr/bin/udevadm"
-	usermodCmd  = "/usr/sbin/usermod"
-	useraddCmd  = "/usr/sbin/useradd"
+	chrootCmd     = "/usr/bin/chroot"
+	groupaddCmd   = "/usr/sbin/groupadd"
+	idCmd         = "/usr/bin/id"
+	mdadmCmd      = "/usr/sbin/mdadm"
+	mountCmd      = "/usr/bin/mount"
+	sgdiskCmd     = "/usr/sbin/sgdisk"
+	udevadmCmd    = "/usr/bin/udevadm"
+	usermodCmd    = "/usr/sbin/usermod"
+	useraddCmd    = "/usr/sbin/useradd"
+	restoreconCmd = "/usr/sbin/restorecon"
```

Comment:
(Sort of random aside but I'm not sure why we're not just searching `$PATH`)

Reply:
I asked around and no one seems to know. I agree it'd be easier to just use `PATH`. @crawford's best guess was to prevent attacks where $PATH is changed, but its running from the initramfs.